### DISCUSSION TITLE
Toggle Darkmode effect from Vuetify3 Doc

### DISCUSSION THREAD

**Discussion by snmabaur** [vuetifyjs/vuetify#16605] [General]
Hi

does anyone know how to use the animation when toggle dark mode is clicked? Only working on Vuetify 3 documentation `https://next.vuetifyjs.com/en/`

It's like a circle starting on the top right corner and getting bigger over the whole page

Thanks for any help :)

**Comment by GreyXor** [comment]
https://github.com/vuetifyjs/vuetify/blob/master/packages/docs/src/App.vue#L57-L137
